src/scripts/OSFramework/OSUI/Pattern/AnimatedLabel/AnimatedLabel.ts

```diff
@@ -60,7 +60,7 @@ namespace OSFramework.OSUI.Patterns.AnimatedLabel {
 
 		// Method that implements the toggle of the state of the input.
 		// It can either add or remove the class "active" of the input.
-		private _inputStateToggle(isFocus: boolean | undefined): void {
+		private _inputStateToggle(isFocus: boolean): void {
```

Comment:
@jfranco09 it was the type I was mention, not the value ;)
The entire line, should be:
private _inputStateToggle(**isFocus = false**): void {

Reply:
But, isn't a problem removing the type notation? 
I'm still learning the typescript ropes, but for my understanding, by removing it, we are allowing the function to accept null and undefined values as input, right?

Reply:
Hi @jfranco09,

I believe undefined and null are primitive values, and so, by default, TypeScript considers null and undefined to be separate types from boolean, and treated as false if used.